docs/guide/apm-getting-started.asciidoc

```diff
@@ -77,9 +77,11 @@ To start APM Server, run:
 
 [source,bash]
 ----------------------------------
-$ ./apm-server -e 
+$ curl -XPOST http://localhost:5601/api/kibana/dashboards/import -H 'Content-type:application/json' -H 'kbn-xsrf:true' -d @./_meta/kibana/default/dashboard/apm-dashboards.json
```

Comment:
It seems the part of starting the apm server went missing. Now only the dashboards are loaded but the server is not started anymore afterwards.